core/gdb_console.c

```diff
@@ -21,23 +21,22 @@
 #include <core/gdb_console.h>
 
 
-static void writeStringToExclusiveGdbCommChannel(const char* pString);
 void WriteStringToGdbConsole(const char* pString)
 {
-    writeStringToExclusiveGdbCommChannel(pString);
+    WriteSizeStringToGdbConsole(pString, mri_strlen(pString));
 }
 
 /* Send the 'O' command to gdb to output text to its console.
 
     Command Format: OXX...
     Where XX is the hexadecimal representation of each character in the string to be sent to the gdb console.
 */
-static void writeStringToExclusiveGdbCommChannel(const char* pString)
+void WriteSizeStringToGdbConsole(const char* pString, size_t length)
```

Comment:
Should be called `WriteSizedStringToGdbConsole()` WriteSized vs WriteSize

Reply:
Done.